docs/website/docs/overview/installation.md

```diff
@@ -31,12 +31,12 @@ Installing `odo` on `amd64` architecture:
 
 1. Download the latest release from the mirror:
 ```shell
-curl -L https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v3.0.0~alpha2/odo-linux-amd64 -o odo
+curl -L https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-amd64 -o odo
```

Comment:
We should use `v3.0.0~beta1` everywhere in this file, as the "latest" link will be fixed soon to point to `v2.5.1` (latest stable release).

Reply:
>We should use v3.0.0~beta1 everywhere in this file, as the "latest" link will be fixed soon to point to v2.5.1 (latest stable release).

Do we have an issue open to track this change?

Reply:
This is an issue for the EXD team - they need to update the link manually. They had created an issue on the Content Gateway to have this done automatically, but this still does not work as expected. See [https://issues.redhat.com/browse/CGW-844](https://issues.redhat.com/browse/CGW-844)